torch/onnx/_internal/fx/function_dispatcher.py

```diff
@@ -180,13 +180,23 @@ def _create_op_overload_to_exporter_key_table() -> Dict[
                 table[op_overload] = op_overload_packet._qualified_op_name
     # TODO(justinchuby): is baddbmm different?
     table[torch.ops.aten.baddbmm.default] = "aten::baddbmm"
+    # TODO(titaiwang): aten::sym_size doesn't have overload,
+    # but works fine as overloadpacket
+    table[torch.ops.aten.sym_size] = "aten::sym_size"
     return table
 
 
 # Dictionary that maps torch.ops.aten.* to exporter look up key; e.g.,
 # _OP_OVERLOAD_TO_EXPORTER_KEY_TABLE[torch.add.Tensor] is "aten::add".
 _OP_OVERLOAD_TO_EXPORTER_KEY_TABLE = _create_op_overload_to_exporter_key_table()
 
+_BUILTIN_OVERLOAD_TO_EXPORTER_KEY_TABLE = {
```

Comment:
I would make public if it is used by other modules.

Reply:
As well as _ATENLIB_XXX?